draft-ietf-ppm-dap.md

```diff
@@ -160,10 +160,23 @@ aggregator.
 
 15:
 
-- Reorganize text for clarity and flow.
+- Specify body of responses to aggregation job GET requests. (#651)
```

Comment:
Is this a breaking change?

Reply:
No, AFAICT it only clarified implicit behavior.